.docignore

```diff
@@ -1 +1,2 @@
 **test**
+CHANGELOG.md
```

Comment:
why is this file gitignored now?

Reply:
We are not adding `CHANGELOG.md` file in docs.oceanprotcol.com. I had removed this file manually. Now, I added it in `.docignore` file.